dashboard/lib/tasks/auto_annotate_models.rake

```diff
@@ -0,0 +1,37 @@
+# NOTE: only doing this in development as some production environments (Heroku)
+# NOTE: are sensitive to local FS writes, and besides -- it's just not proper
+# NOTE: to have a dev-mode tool do its thing in production.
+if Rails.env.development?
+  task :set_annotation_options do
+    # You can override any of these by setting an environment variable of the
+    # same name.
+    Annotate.set_defaults({
+      'position_in_routes'      => "before",
+      'position_in_class'       => "before",
+      'position_in_test'        => "before",
+      'position_in_fixture'     => "before",
+      'position_in_factory'     => "before",
+      'position_in_serializer'  => "before",
+      'show_foreign_keys'       => "true",
+      'show_indexes'            => "true",
+      'simple_indexes'          => "false",
+      'model_dir'               => "app/models",
+      'include_version'         => "false",
+      'require'                 => "",
+      'exclude_tests'           => "true",
+      'exclude_fixtures'        => "true",
+      'exclude_factories'       => "false",
+      'exclude_serializers'     => "false",
+      'ignore_model_sub_dir'    => "false",
+      'skip_on_db_migrate'      => "false",
```

Comment:
Correct, annotate runs on "rake db:migrate" in development mode only (which I verified)